index.js

```diff
@@ -45,6 +49,61 @@ app.post('/v1/chat/completions', async (req, res) => {
   }
 });
 
+// Forward /v1/responses to OpenAI (supports streaming)
+app.post("/v1/responses", async (req, res) => {
+  try {
+    const response = await fetch("https://api.openai.com/v1/responses", {
+      method: "POST",
+      headers: {
+        "Authorization": `Bearer ${process.env.OPENAI_API_KEY}`,
```

Comment:
We already have this as a variable on line 12
```suggestion
        "Authorization": `Bearer ${OPENAI_API_KEY}`,
```